src/examples/tutorial/ascent_intro/notebooks/11_ascent_path_formatting.ipynb

```diff
@@ -0,0 +1,420 @@
+   "source": [
+    "# We expect our cycle trigger to render on cycles 300, 600, and 900\n",
+    "# These cycles coorespond with the times 1.0, 2.5, and 4.0\n",
+    "\n",
+    "# Recall that we rendered scenes to three different paths to demonstrate different\n",
+    "#formats that can be passed to the ascent string formatter:\n",
+    "#     - Default Acent formatting:   ``out_fmt_time_trigger-default_{time}``\n",
+    "#     - Float Ascent formatting:    ``out_fmt_time_trigger-float_{time:05.1f}``\n",
+    "#     - Integer Ascent formatting:  ``out_fmt_time_trigger-int_{time:04d}``\n",
+    "\n",
+    "! ls out_fmt_time_trigger*.png"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "354b0c9d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# cleanup any old results\n",
+    "!./cleanup.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a5be45f1",
+   "metadata": {},
+   "source": [
+    "## Path Formatting Example 3\n",
+    "### Using the ``family`` keyword\n",
+    "\n",
+    "The family value is a unique value based on existing files in the output directory.\n",
+    "\n",
+    "It is used to avoid overwriting existing files in the output directory with a\n",
+    "matching output file pattern, including those from previous runs of Ascent. It is set to one\n",
+    "greater than the maximum detected family value of the matching file pattern, or zero if no\n",
+    "matching files are found.\n",
+    "\n",
+    "``Default Format: 06d``\n",
+    "\n",
+    "For this example, we will only be outputting to one path. You can run the example multiple times\n",
+    "to see how the family value behavior persists across runs of ascent.\n",
+    "  - Output image prefix:   ``out_fmt_family_{family:03d}``"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50e97c80",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use Ascent to bin an input mesh in a few ways\n",
+    "a = ascent.Ascent()\n",
+    "\n",
+    "# open ascent\n",
+    "a.open()\n",
+    "\n",
+    "# publish mesh to ascent\n",
+    "a.publish(mesh)\n",
+    "\n",
+    "# setup actions\n",
+    "actions = conduit.Node()\n",
+    "add_act = actions.append()\n",
+    "add_act[\"action\"] = \"add_scenes\"\n",
+    "\n",
+    "# declare two scenes (s1 and s2) to render the dataset\n",
+    "scenes = add_act[\"scenes\"]\n",
+    "\n",
+    "scenes[\"s1/plots/p1/type\"] = \"pseudocolor\"\n",
+    "scenes[\"s1/plots/p1/field\"] = \"braid\"\n",
+    "scenes[\"s1/image_prefix\"] = \"out_fmt_family_{family:03d}\"\n",
+    "\n",
+    "# view our full actions tree\n",
+    "print(actions.to_yaml())\n",
+    "\n",
+    "# Run ascent a couple times to demonstrate the family value behavior\n",
+    "for _ in range(5):    \n",
+    "    # execute the actions\n",
+    "    a.execute(actions)\n",
+    "\n",
+    "# retrieve the info node that contains the trigger and query results\n",
```

Comment:
We aren't really using ascent triggers here, so we can simplify the descriptions